samples/Mvc.RenderViewToString/Views/EmailPartial.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿@inject Mvc.RenderViewToString.EmailReportHelper ReportHelper
+
+    This is the partial view content.<br />
+    @ReportHelper.GenerateReport(Model.UserData1, Model.UserData2).<br />
```

Comment:
Would a `ViewComponent` work here? If so, that would show a bit more working e2e.